Plan/src/main/java/com/djrapitops/plan/system/info/VelocityInfoSystem.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * License is provided in the jar as LICENSE also here:
+ * https://github.com/Rsl1122/Plan-PlayerAnalytics/blob/master/Plan/src/main/resources/LICENSE
+ */
+package com.djrapitops.plan.system.info;
+
+import com.djrapitops.plan.api.exceptions.ParseException;
+import com.djrapitops.plan.api.exceptions.connection.NoServersException;
+import com.djrapitops.plan.api.exceptions.connection.WebException;
+import com.djrapitops.plan.system.info.connection.VelocityConnectionSystem;
+import com.djrapitops.plan.system.info.request.CacheRequest;
+import com.djrapitops.plan.system.info.request.GenerateInspectPageRequest;
+import com.djrapitops.plan.system.info.request.GenerateInspectPluginsTabRequest;
+import com.djrapitops.plan.system.info.request.InfoRequest;
+import com.djrapitops.plan.system.info.server.ServerInfo;
+import com.djrapitops.plan.system.webserver.cache.PageId;
+import com.djrapitops.plan.system.webserver.cache.ResponseCache;
+import com.djrapitops.plan.system.webserver.response.errors.InternalErrorResponse;
+import com.djrapitops.plan.system.webserver.response.pages.NetworkPageResponse;
+
+/**
+ * InfoSystem for Velocity.
+ *
+ * Based on BungeeInfoSystem
+ *
+ * @author MicleBrick
+ */
+public class VelocityInfoSystem extends InfoSystem {
```

Comment:
Duplication of this class is unnecessary, as only benefit is having it in velocity specific class.
This can be sorted later by refactoring common classes into network specific names